src/__tests_/App.test.tsx

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import { render, fireEvent, cleanup } from 'test-utils';
```

Comment:
```suggestion
import { render, cleanup } from 'test-utils';
```
nit: remove unused dependencies/imports - we'll eventually have a eslint rule that checks for this